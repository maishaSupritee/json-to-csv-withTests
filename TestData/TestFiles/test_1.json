{
  "fruit": [
    {
      "C": "",
      "A": ""
    },
    {
      "T": "",
      "S": ""
    }
  ]
}
