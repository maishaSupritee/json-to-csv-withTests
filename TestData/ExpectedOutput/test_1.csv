"fruit_C","fruit_A","fruit_T","fruit_S"

"","","",""

"","","",""

